specification/portal/resource-manager/readme.md

```diff
@@ -122,16 +123,26 @@ require: $(this-folder)/../../../profiles/readme.md
 # all the input files across all versions
 input-file:
   - $(this-folder)/Microsoft.Portal/preview/2019-01-01-preview/portal.json
+  - $(this-folder)/Microsoft.Portal/preview/2019-01-01-preview/tenantConfiguration.json
   - $(this-folder)/Microsoft.Portal/preview/2018-10-01-preview/portal.json
   - $(this-folder)/Microsoft.Portal/preview/2015-08-01-preview/portal.json
 
 ```
 
-If there are files that should not be in the `all-api-versions` set, 
+If there are files that should not be in the `all-api-versions` set,
 uncomment the  `exclude-file` section below and add the file paths.
 
 ``` yaml $(tag) == 'all-api-versions'
 #exclude-file: 
 #  - $(this-folder)/Microsoft.Example/stable/2010-01-01/somefile.json
 ```
 
+## Suppression
```

Comment:
Please go through Swagger Suppression Process https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/85/Swagger-Suppression-Process?anchor=obtaining-suppression

Reply:
It's a warning and won't block the CI , no needs to suppress.

Reply:
> It's a warning and won't block the CI , no needs to suppress.

I'm using OpenAPI Hub (the tool was recommended in the API docs) and that tool doesn't pass validation without suppress (formally these suppress lines were added by tool itself, not manually).

Reply:
@AKachan0v  In this case please revert this change